android/settings.gradle

```diff
@@ -7,4 +7,6 @@ project(':reactnativenotifications').projectDir = new File(rootProject.projectDi
 include ':reactnativekeyboardinput'
 project(':reactnativekeyboardinput').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-keyboard-input/lib/android')
 apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
+include ':rn-user-defaults'
```

Comment:
You can give it a shoot, but I think you don't have to manually link this lib (build.gradle, MainApp and here).
RN does it automatically since 0.60.
Try removing your changes from these files and running again.

Reply:
Was just following the recommendations on the `rn-user-dafaults` `README.md` file pointed out by Alexandre! Good to know. Let me change it. Thx!

Reply:
Changed!